src/main/java/frc/robot/commands/DefaultIntakeCommand.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.IntakeConstants;
+import frc.robot.subsystems.Input;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class DefaultIntakeCommand extends Command {
+
+  private IntakeSubsystem mIntake;
+  /** Creates a new DefaultIntakeCommand. */
+  public DefaultIntakeCommand(IntakeSubsystem intake) {
+    mIntake = intake;
+    addRequirements(intake);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    mIntake.stopMotor();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if (mIntake.pieceInIntake()) {
+      mIntake.stopMotor();
+    }
+    if (Input.getLeftBumperXbox() && !mIntake.pieceInIntake()) {
```

Comment:
Can you add an override feature to this? Last year we had an "Ultra Instinct" mode on the arm which essentially removed soft limits. In case something goes wrong could you create a constant that can be changed with a button that skips the pieceInIntake requirement? If you want, you can call it something DBZ related lol.